Tests/LibWeb/Text/input/WebAudio/offline-audiocontext-renderquantum.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<script>
+    test(() => {
+        const audioContext = new OfflineAudioContext(1, 1, 44100)
+        println(`${audioContext.renderQuantumSize}`);
```

Comment:
Instead of having a single test for a single attribute, maybe you could test the other OfflineAudioContext attributes here as well? Makes it a bit more useful going forward.

Reply:
Right, that makes sense then. You could add a comment to the test explaining how it could probably be removed as soon as we can import the updated WPT ctor test.